Flickr/Sources/FlickrAppVCs/FavoritesVC.swift

```diff
@@ -15,6 +16,17 @@ class FavoritesVC: UIViewController {
     private weak var textView: UILabel!
     private var isFavouritesEmpty: Bool?
     private var favoritesArray: [FavoriteImageData]?
+    var dbPool: DatabasePool
+
+    init(dbPool: DatabasePool) {
+        self.dbPool = dbPool
+        super.init(nibName: nil, bundle: nil)
+    }
```

Comment:
Please avoid having reference to dbpool in VC. If you need access to persisted data then you can reach out to coordinator using delegates and get data from the persistencemanager and update the UI.